frontend/src/styles.scss

```diff
@@ -9,18 +9,32 @@
   font-weight: bold;
 }
 
-@font-face {
-  font-family: 'Roboto Regular';
-  src: url('./assets/fonts/Roboto-Regular.ttf');
-  font-weight: normal;
-}
-
 @font-face {
   font-family: 'Roboto Medium';
   src: url('./assets/fonts/Roboto-Medium.ttf');
   font-weight: 500;
 }
 
+@font-face {
+  font-family: 'Roboto Regular';
+  src: url('./assets/fonts/Roboto-Regular.ttf');
+  font-weight: normal;
+}
+
 html, body { height: 100%; }
```

Comment:
also please add https://cssreset.com/scripts/eric-meyer-reset-css/ as separate file and import here, 
and these styles also can be removed.